js/animation.js

```diff
@@ -0,0 +1,321 @@
+//slider catalog
+{
+    class SliderCatalog {
+        constructor(flag) {
+            this.plusCat = document.querySelector('.plusCat');
+            this.catDescr = document.querySelector('.catDescr');
+            this.flag = flag;
+            this.catDescr.classList.add('p-show');
+            this.catDescr.style.display = 'none';
+        }
+
+        showCatalog(disp, transform, colour, opac, flag) {
+            this.catDescr.style.display = disp;
+            this.plusCat.style.transform = transform;
+            this.plusCat.style.color = colour;
+            this.plusCat.style.opacity = opac;
+            this.flag = flag;
+            return true;
+        }
+
+        showCat = () => {
```

Comment:
```suggestion
        showCat() {
```
Is it a method or a field?